### PR TITLE
feat(environments): #591 form to manually add releases as alternative to curl commands

### LINKED ISSUE #591: Manual deploy in new design
Dependent on #800 

An alternative to using the curl command, the form can be used for manual releases.

Form on the release list page (similar to the token list page) with these fields:

- version (string)
- token (dropdown of their tokens for that environment)
- state (startBuild...finishDeploy - depending on their environment type)

### PR DESCRIPTION
**Pull request by chowbertilla**
closes #591 

- Submit data
- styling (similar to Token form)

### Notes

A summary of what was achieved in this PR

- [ ] API: Task One (eg. updated `deployed` model )
- [ ] UI: Task Three (eg. added validation rules on create environment page)
- [ ] Documentation: Updated accordingly

### CODE REVIEW

**Comment by chowbertilla** [comment]
@eddiejaoude We still need to create the submit button

**Comment by eddiejaoude** [comment]
@chowbertilla I updated the branch with the form styling similar to the tokens form and added the button. There is still the form submission (action) that needs doing, lets pair on that together so you can see how to post the data from the form to the backend API 🤓

File: web/src/app/environments/releases/release.service.ts [hunk 1]
```diff
@@ -11,4 +11,10 @@ export class ReleaseService {
   findAllByEnvironmentId(environmentId: string): Observable<List<Release>> {
     return this.http.get<List<Release>>(`{api}/environments/${environmentId}/releases`);
   }
+
+  add(environmentId: string, release: Release, state: string): Observable<Release> {
+    return this.http.post<Release>(`{api}/environments/${environmentId}/deployed/${release.token.id}/${state}`, {
+      release: release.version
```
Comment (eddiejaoude) [inline web/src/app/environments/releases/release.service.ts]: Missing ending semi colon

Comment (eddiejaoude) [reply web/src/app/environments/releases/release.service.ts]: I am fixing the API bug you found yesterday, and I will add it to this PR, I will add the missing semi colon above 🤓

Comment (eddiejaoude) [reply web/src/app/environments/releases/release.service.ts]: Remind me to export my IntelliJ settings, so the IDE highlights the lint errors

**Pull request merged**